kratos/python/add_utilities_to_python.cpp

```diff
@@ -739,28 +740,97 @@ void AddUtilitiesToPython(pybind11::module& m)
+        ;
+
+    py::class_<TimeDiscretization::Newmark>(m, "Newmark")
+        .def(py::init<>())
+        .def(py::init<const double, const double>())
+        .def("GetBeta", &TimeDiscretization::Newmark::GetBeta)
+        .def("GetGamma", &TimeDiscretization::Newmark::GetGamma)
+        ;
+    py::class_<TimeDiscretization::Bossak>(m, "Bossak")
+        .def(py::init<>())
+        .def(py::init<const double>())
+        .def(py::init<const double, const double, const double>())
+        .def("GetBeta", &TimeDiscretization::Bossak::GetBeta)
+        .def("GetGamma", &TimeDiscretization::Bossak::GetGamma)
+        .def("GetAlphaM", &TimeDiscretization::Bossak::GetAlphaM)
+        ;
+    py::class_<TimeDiscretization::GeneralizedAlpha>(m, "GeneralizedAlpha")
+        .def(py::init<>())
+        .def(py::init<const double, const double>())
+        .def(py::init<const double, const double, const double, const double>())
+        .def("GetBeta", &TimeDiscretization::GeneralizedAlpha::GetBeta)
+        .def("GetGamma", &TimeDiscretization::GeneralizedAlpha::GetGamma)
+        .def("GetAlphaM", &TimeDiscretization::GeneralizedAlpha::GetAlphaM)
+        .def("GetAlphaF", &TimeDiscretization::GeneralizedAlpha::GetAlphaF)
+        ;
+
+    std::size_t (*GetMinimumBufferSizeBDF1)(const TimeDiscretization::BDF1&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBDF2)(const TimeDiscretization::BDF2&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBDF3)(const TimeDiscretization::BDF3&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBDF4)(const TimeDiscretization::BDF4&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBDF5)(const TimeDiscretization::BDF5&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBDF6)(const TimeDiscretization::BDF6&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeNewmark)(const TimeDiscretization::Newmark&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeBossak)(const TimeDiscretization::Bossak&) = &TimeDiscretization::GetMinimumBufferSize;
+    std::size_t (*GetMinimumBufferSizeGneralizedAlpha)(const TimeDiscretization::GeneralizedAlpha&) = &TimeDiscretization::GetMinimumBufferSize;
+
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF1 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF2 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF3 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF4 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF5 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBDF6 );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeNewmark );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeBossak );
+    m.def("GetMinimumBufferSize", GetMinimumBufferSizeGneralizedAlpha );
```

Comment:
you have a typo here (i'll accept anyway)

Reply:
oh, thx for spotting
I will fix in the next PR, thanks for taking a look!